dasharo-stability/capsule-update.robot

```diff
@@ -278,3 +342,71 @@ Display Preparation Instructions
     Log To Console    Without it, a successful flash of DUT will prevent tests from working
     Log To Console    correctly.
     Log To Console    \n******************************************************************************
+
+Prepare For Logo Persistence Test
+    Log To Console    PREPARE: Logo Persistence Test
+
+    Run    rm -rf dcu
+    Run    git clone https://github.com/Dasharo/dcu
+    Run    cp ${FW_FILE} dcu/coreboot.rom
+
+    ${out}=    Run
+    ...    cbfstool ./dcu/coreboot.rom layout -w | grep BOOTSPLASH > /dev/null && echo "Custom logo supported" || echo "Custom logo not supported"
```

Comment:
This also depends on `cbfstool`, but I think we can skip it. The same test is done by `dcu`: https://github.com/Dasharo/dcu/blob/main/src/logo_command.sh#L20, it should be enough to test for `Customizable logo not supported` in the output.

Reply:
yes but we have to check if the logo persists across the capsule update... so it is not actually the same.

Reply:
OK, then what is the difference?

Reply:
We are checking if the logo was not changed by the update.

Reply:
Here `cbfstool` is used to check whether custom logo is supported.  Seems like you can just use `${CUSTOM_LOGO_SUPPORT}` for that.

Reply:
Removed this check in: 94c3fd7c4f176ca7c53085393f1b9e90b19b8a50

Reply:
But now it doesn't change the logo, it just clones the repo and copies the file without any modification.

Reply:
Ive cut too much of the code. Fixed in: 3e278bdb6a3fbe17c6ebfb35e06ad28c899525dc